herbert_core/admin/@opt_config_manager/private/find_comp_type_.m

```diff
@@ -60,6 +61,12 @@
         n_profile=n_profile+1;
     end
     pc_type = types{n_profile};
+
+end
+
+is_jenkins_true = is_jenkins()
```

Comment:
why are you evaluating `is_jenkins()` here and on the next line ... is this variable needed?

Reply:
Ah thanks, this was leftover from debugging